terraform/fec_rds.tf

```diff
@@ -16,6 +16,61 @@ provider "aws" {
   region = "${var.region}"
 }
 
+# logging role
+aws_iam_role_policy = <<EOF
```

Comment:
The syntax here should be:

```hcl
resource "aws_iam_role_policy" "test_policy" {
  name = "test_policy"
  role = "${aws_iam_role.test_role.id}"
  policy = <<EOF
...
EOF
}
```

The docs at https://www.terraform.io/docs/providers/aws/r/iam_role_policy.html include a good example. You can also use the `terraform validate` command to check whether the configuration is valid.